test/e2e/framework/pod/create.go

```diff
@@ -223,33 +215,47 @@ func MakePodSpec(podConfig *Config) *v1.PodSpec {
 		podSpec.SecurityContext.FSGroupChangePolicy = podConfig.PodFSGroupChangePolicy
 	}
 
+	setVolumes(podSpec, podConfig.PVCs, podConfig.InlineVolumeSources, podConfig.PVCsReadOnly)
+
+	if runtime.GOOS != "windows" {
+		podSpec.SecurityContext.SELinuxOptions = podConfig.SeLinuxLabel
+	}
```

Comment:
This assignment to `podSpec.SecurityContext.SELinuxOptions` seems unrelated to this PR. Should it be included?

Reply:
Right, this has been removed in d9c4d8bff833e288a9b366b3b4b69920e35aaaca.